tools/FileTestDefs.sh

```diff
@@ -181,7 +181,8 @@ long_lines_exclude() {
     grep -Ev 'https?://' | \
         grep -v '// IWYU pragma:' | \
         grep -v '// NOLINT' | \
-        grep -v '\\snippet'
+        grep -v '\\snippet' | \
+        grep -v '\\image'
```

Comment:
It looks like someone (probably me) was sloppy with the last addition here.  Could you add `\snippet` and `\image` to `long_lines_test` below.